s3/src/main/scala/akka/stream/alpakka/s3/S3Settings.scala

```diff
@@ -67,7 +69,7 @@ object S3Settings {
   /**
    * Scala API: Creates [[S3Settings]] from a [[com.typesafe.config.Config Config]] object.
    */
-  def apply(config: Config, configurationPrefix: String = "akka.stream.alpakka.s3"): S3Settings = {
+  def apply(config: Config, configurationPrefix: String = configPath): S3Settings = {
```

Comment:
We need to agree on a pattern here. In other connectors passing a config expects it to read from the resolved section. Passing ActorSystem reads from the configPath.